tools/macos-installer/pkgbuild/npm/scripts/postinstall

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cd /usr/local/bin || exit 1
+ln -sf ../lib/node_modules/npm/bin/npm-cli.js npm
+ln -sf ../lib/node_modules/npx/bin/npx-cli.js npx
```

Comment:
This should be:

```sh
ln -sf ../lib/node_modules/npm/bin/npx-cli.js npx
```

Reply:
Ok to push those changes to same commit, or should I add new commits, and squash later?

Reply:
Everything will be squashed when it lands in master, so you can just add new commits.